src/controllers/auth.controller.ts

```diff
@@ -10,10 +10,16 @@ import {
 import { getRepository } from 'typeorm';
 import * as _ from 'lodash';
 import { User } from '@entities/user.entity';
-import { createJWT } from '@services/jwt.service';
+import JWTService from '@services/jwt.service';
 
 @JsonController('/auth')
 export class AuthController {
+  jwt: JWTService;
+
+  constructor(private jwtService: JWTService) {
```

Comment:
I think only adding it to the constructor will work, and you can access it as `this.jwtService` can you verify that?